packages/api/src/client/internal/generated/openapi/namespaces/OntologiesV2.ts

```diff
@@ -0,0 +1,971 @@
+    return _request(
+        "getObjectSetV2",
+        "GET",
+        "/v2/ontologies/{ontology}/objectSets/{objectSetRid}",
+        __undefined,
+        __undefined,
+        __undefined,
+        {
+            ontology,
+            objectSetRid,
+        },
+        __undefined,
+        "application/json",
+    );
+}
+
+/**
+ * Load the ontology objects present in the `ObjectSet` from the provided object set definition.        
+ *
+ * Third-party applications using this endpoint via OAuth2 must request the following operation scope: `api:read-data`.
+ *
+ */
+export function loadObjectSetV2<TResponse>(_request: OpenApiRequest<LoadObjectSetResponseV2, TResponse>, ontology: OntologyApiName, request: LoadObjectSetRequestV2): Promise<TResponse> {
+    return _request(
+        "loadObjectSetV2",
+        "POST",
+        "/v2/ontologies/{ontology}/objectSets/loadObjects",
+        request,
+        __undefined,
+        __undefined,
+        {
+            ontology,
+        },
+        "application/json",
+        "application/json",
+    );
+}
+
+/**
+ * Aggregates the ontology objects present in the `ObjectSet` from the provided object set definition.        
+ *
+ * Third-party applications using this endpoint via OAuth2 must request the following operation scope: `api:read-data`.
+ *
+ */
+export function aggregateObjectSetV2<TResponse>(_request: OpenApiRequest<AggregateObjectSetResponseV2, TResponse>, ontology: OntologyApiName, request: AggregateObjectSetRequestV2): Promise<TResponse> {
+    return _request(
+        "aggregateObjectSetV2",
+        "POST",
+        "/v2/ontologies/{ontology}/objectSets/aggregate",
+        request,
+        __undefined,
+        __undefined,
+        {
+            ontology,
+        },
+        "application/json",
+        "application/json",
+    );
+}
+
+/**
+ * Fetches a list of the available model deployments within a given Ontology.
+ *
+ */
+export function listDeployments<TResponse>(_request: OpenApiRequest<ListDeploymentsResponse, TResponse>, ontology: OntologyApiName): Promise<TResponse> {
+    return _request(
+        "listDeployments",
+        "GET",
+        "/v2/ontologies/{ontology}/models/deployments",
+        __undefined,
+        __undefined,
+        __undefined,
+        {
+            ontology,
+        },
+        __undefined,
+        "application/json",
+    );
+}
+
+/**
+ * Fetches information about a model deployment within a given Ontology.
+ *
+ */
+export function getDeployment<TResponse>(_request: OpenApiRequest<DeploymentMetadata, TResponse>, ontology: OntologyApiName, deployment: DeploymentApiName): Promise<TResponse> {
+    return _request(
+        "getDeployment",
+        "GET",
+        "/v2/ontologies/{ontology}/models/deployments/{deployment}",
+        __undefined,
+        __undefined,
+        __undefined,
+        {
+            ontology,
+            deployment,
+        },
+        __undefined,
+        "application/json",
```

Comment:
Given 99% of our APIs all expect json in return and if they have a body, send JSON, can we just make these the defaults for our custom request function? This lets these be omitted and slim down the code quite a bit.